locales/templates/app.pot

```diff
@@ -933,13 +938,14 @@ msgctxt "survey_notification_description"
 msgid "Shape a better product for Indonesia"
 msgstr ""
 
-#. Snack bar message indicates there's a new tab loaded in the background
+#. A message that is shown for a few seconds that indicates there's a new tab
+#. loaded in the background.
 msgctxt "new_background_tab_hint"
 msgid "New tab opened"
 msgstr ""
 
-#. Snack bar button that switch the newly loaded background tab to the
-#. foreground
+#. An action shown with new_background_tab_hint that gives user an option to
```

Comment:
Same as for screenshot_saved --> So more something like "An action shown in a temporary notification that gives user etc)